src/Shipwreck.Phash/ImagePhash.cs

```diff
@@ -73,11 +74,56 @@ public static Digest ComputeDigest(string fileName, double sigma = DEFAULT_SIGMA
             }
         }
 
-        public static Digest ComputeDigest(Stream fileStream, double sigma = DEFAULT_SIGMA, double gamma = DEFAULT_GAMMA, int numberOfAngles = DEFAULT_NUMBER_OF_ANGLES)
+        public static Digest ComputeDigest(Stream bitmapStream, double sigma = DEFAULT_SIGMA, double gamma = DEFAULT_GAMMA, int numberOfAngles = DEFAULT_NUMBER_OF_ANGLES)
         {
-            var bf = BitmapFrame.Create(fileStream, BitmapCreateOptions.None, BitmapCacheOption.OnLoad);
+            var bf = BitmapFrame.Create(bitmapStream, BitmapCreateOptions.None, BitmapCacheOption.OnLoad);
 
-            return ComputeDigest(bf.ToByteImageOfYOrB(), sigma, gamma, numberOfAngles: numberOfAngles);
+            return ComputeDigest(bf, sigma, gamma, numberOfAngles: numberOfAngles);
+        }
+
+        public static Digest ComputeDigest(BitmapSource bitmapSource, double sigma = DEFAULT_SIGMA, double gamma = DEFAULT_GAMMA, int numberOfAngles = DEFAULT_NUMBER_OF_ANGLES)
```

Comment:
Change method name like `ComputeBitmapDigest` to avoid PresentationCore dependent signature to be included in method group. Such overload forces non-WPF user to add references explicitly.